### ISSUE TITLE
"text-align: center" not followed

### ISSUE THREAD

**Issue by yuklai** [danomatic/react-pdf-html#22]
The following html should render the text center aligned. However, when generating the using react-pdf-html, they are not. 

Have you seen this issue before?

```
<html>
  <head>
    <style>
    div {
      text-align: center; 
      font-size: 20px
    }
    
    .page {
      width: 8.5in;
      height: 11in;
      display: flex;
      justify-content: center;
      align-items: center; 
    }
    </style>
  </head>

  <body>
    <div class="page">
      <div>
        <div>ABC</div>
        <div>12345678</div>
      </div>
    </div>
  </body>
</html>
```

![image](https://user-images.githubusercontent.com/565959/142967173-22a7e748-f1f1-4d1f-b31c-2911545e5040.png)

**Comment by danomatic** [comment]
Thank you for reporting this.

First of all, I just tried your to recreate your with only react-pdf components:
```
        <View debug style={{
          display: 'flex',
          justifyContent: 'center',
          alignItems: 'center'
        }}>
          <View debug style={{
            textAlign: 'center',
            padding: 20
          }}>
            <View>
              <Text>ABC</Text>
            </View>
            <View>
              <Text>12345678</Text>
            </View>
          </View>
        </View>
      </Page>
```
For me, this rendered the text along the right edge of the document. I couldn't figure it out, but then I updated `@react-pdf/renderer` from version `2.0.12` to `2.0.21` and it properly rendered the text in the center of the page. If you are not using the latest version of `@react-pdf/renderer`, you might want to update.

Second of all, you have wrapped your content in `html` and `body` tags. By default those are rendered as block level elements, similar to how a `div` tag would render. If you remove those wrappers, I think you will get the vertical centering behavior that you want.
